src/main/scala/firrtl/transforms/Flatten.scala

```diff
@@ -0,0 +1,113 @@
+// See LICENSE for license details.
+
+package firrtl
+package passes
+
+import firrtl.ir._
+import firrtl.Mappers._
+import firrtl.annotations._
+import scala.collection.mutable
+
+// Tags an annotation to be consumed by this pass
```

Comment:
All comments around class and function declarations should by ScalaDoc style, see http://docs.scala-lang.org/style/scaladoc.html.